docker-app/qfieldcloud/urls.py

```diff
@@ -52,8 +54,8 @@
         schema_view.with_ui("swagger", cache_timeout=0),
         name="schema-swagger-ui",
     ),
+    path(os.environ.get("QFIELDCLOUD_ADMIN_URI", "admin/"), admin.site.urls),
```

Comment:
I think this is quite straight forward to catch and without the admin in a critical situation it could be disaster, especially with all the k8s stuff. I do agree it would be harder to find that the var is missing, but my thought was safety first.